lib/mayaUsd/python/wrapAdaptor.cpp

```diff
@@ -250,19 +250,8 @@ static bool _AttributeAdaptor_Set(UsdMayaAttributeAdaptor& self, const VtValue&
 
 static std::string _AttributeAdaptor__repr__(const UsdMayaAttributeAdaptor& self)
 {
-    std::string                  schemaName;
```

Comment:
Any reason to no longer print the schema name? Not that I care that much...

Reply:
It's because there's no way to access it from the new UsdMayaAttributeAdaptor since that only stores the attribute definition and not which schema/prim definition the attribute came from. 

Since the "old" UsdMayaAttributeAdaptor was holding an SdfAttributeSpec as the attribute definition, the code was sussing out the schema name for the spec's path, but that was making use of an implementation detail that we make no guarantee will be correct. There's no requirement that the property spec is a child of a prim with the schema's name, and we've recently added some property definition composition behaviors could cause the schema name returned by this method to be wrong.